tools/keygen.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+const uuidgen = require('node-uuid-generator');
+
+if (process.argv.length <= 2) {
+  console.log('Usage: ' + __filename + ' <secret string>');
+  process.exit(-1);
+}
+
+var secret = process.argv[2];
+
+const userid = uuidgen.generate();
+const token = jwt.sign({ id: userid }, secret);
```

Comment:
We should also store this userid with a password in the table so that the user can use the /token endpoint

Reply:
I'm goign to manually enter the APIUsers for now.